eq_translations/extractable_strings.py

```diff
@@ -7,6 +7,7 @@
     {"json_path": "$.submission.title", "description": "Submission title"},
     {"json_path": "$.submission.warning", "description": "Submission warning"},
     {"json_path": "$.sections[*].title", "description": "Section title"},
+    {"json_path": "$..page_title", "description": "Custom page title"},
```

Comment:
This should probably just be `Page title` as I don't think the fact that it is custom will be relevant to a translator.